theatre/studio/src/uiComponents/PointerEventsHandler.tsx

```diff
@@ -68,6 +68,10 @@ const PointerEventsHandler: React.FC<{
   )
 }
 
+/**
+ * Question: Is this referring to the user's pointer input device?
+ * Or is this locking related to the playhead "cursor"?
```

Comment:
Oh it just sets `body.style.cursor = cursor`, which is the "lock". Maybe `useCssCursorLock()` is a better name?